core/src/main/java/ai/timefold/solver/core/impl/solver/DefaultSolutionManager.java

```diff
@@ -113,12 +116,20 @@ public ScoreAnalysis<Score_> analyze(Solution_ solution, ScoreAnalysisFetchPolic
         return analysis;
     }
 
+    @Override
+    public <In_, Out_> List<RecommendedAssignment<Out_, Score_>> recommendAssignment(Solution_ solution,
```

Comment:
I believe we need to add a new overloaded method without `fetchPolicy` param as we are using such a method in the documentation.

Reply:
It's a default method on `SolutionManager`.